eclair-core/src/main/scala/fr/acinq/eclair/io/Peer.scala

```diff
@@ -389,6 +730,19 @@ class Peer(val nodeParams: NodeParams,
     // let's bring existing/requested channels online
     channels.values.toSet[ActorRef].foreach(_ ! INPUT_RECONNECTED(connectionReady.peerConnection, connectionReady.localInit, connectionReady.remoteInit)) // we deduplicate with toSet because there might be two entries per channel (tmp id and final id)
 
+    // We tell our peer what our current feerates are.
+    connectionReady.peerConnection ! nodeParams.recommendedFeerates(remoteNodeId, nodeParams.currentFeerates, connectionReady.localInit.features, connectionReady.remoteInit.features)
+
+    if (nodeParams.features.hasFeature(Features.FundingFeeCredit) && connectionReady.remoteInit.features.hasFeature(Features.FundingFeeCredit)) {
```

Comment:
We should use local features, and the `Features.canUseFeature` helper:

```suggestion
    if (Features.canUseFeature(connectionReady.localInit.features, connectionReady.remoteInit.features, Features.FundingFeeCredit)) {
```